zip-0252.rst

```diff
@@ -2,7 +2,185 @@
 
   ZIP: 252
   Title: Deployment of the NU5 Network Upgrade
-  Owners: Daira Hopwood <daira@electriccoin.co>
-  Status: Reserved
+  Owners: Daira Hopwood <daira@electriccoin.co>, teor <teor@zfnd.org>
+  Status: Draft
   Category: Consensus / Network
-  Discussions-To: <https://github.com/zcash/zips/issues/397>
+  Created: 2021-02-23
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be
+interpreted as described in RFC 2119. [#RFC2119]_
+
+The term "network upgrade" in this document is to be interpreted as described in
+ZIP 200. [#zip-0200]_
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in
+section 3.11 of the Zcash Protocol Specification [#protocol-networks]_.
+
+NU5 is the code-name for the sixth Zcash network upgrade, also known as
+Network Upgrade 5.
+
+
+Abstract
+========
+
+This proposal defines the deployment of the NU5 network upgrade.
+
+
+Specification
+=============
+
+NU5 deployment
+-----------------
+
+The primary sources of information about NU5 consensus protocol changes are:
+
+- The Zcash Protocol Specification [#protocol]_
+- ZIP 200: Network Upgrade Mechanism [#zip-0200]_
+- **TODO: Select ZIPs**
+
+The network handshake and peer management mechanisms defined in ZIP 201 [#zip-0201]_
+also apply to this upgrade.
+
+
+The following network upgrade constants [#zip-0200]_ are defined for the NU5
+upgrade:
+
+CONSENSUS_BRANCH_ID
+  ``0xF919A198``
+
+
+ACTIVATION_HEIGHT (NU5)
+  Testnet: 1382200
+
+  Mainnet: 1345400
```

Comment:
```suggestion
  Mainnet: TODO
```

The heights should only be set in the ZIP when the actual decisions (for Testnet and for Mainnet) are made.